tests/helpers/toLowerCase.test.js

```diff
@@ -1,11 +0,0 @@
-var helpers = require('../../js/main.js');
```

Comment:
I think I liked better having the tests in separate files. it was very easy to find them